app/javascript/packs/banner_editor.jsx

```diff
@@ -180,19 +196,26 @@ export default class BannerEditor extends React.Component {
   async fetchBanner(){
 
     let that = this
+    let url;
+
+    if(this.state.defaultSiteBannerMode){
+      url = '/publishers/' + document.getElementById("publisher_id").value + "/site_banners/" + this.props.defaultSiteBanner.id + "?default=" + this.state.defaultSiteBannerMode
+    }
+    else{
+      url = '/publishers/' + document.getElementById("publisher_id").value + "/site_banners/" + this.props.channelBanners[this.state.channelIndex].id;
```

Comment:
This would change if we replace the `map` function i mentioned about with `site_banner_id`